browser/brave_wallet/brave_wallet_tab_helper.cc

```diff
@@ -23,7 +24,10 @@
 namespace brave_wallet {
 
 BraveWalletTabHelper::BraveWalletTabHelper(content::WebContents* web_contents)
-    : content::WebContentsUserData<BraveWalletTabHelper>(*web_contents) {}
+    : content::WebContentsUserData<BraveWalletTabHelper>(*web_contents) {
+  brave_wallet::WalletNotificationServiceFactory::GetServiceForContext(
+      web_contents->GetBrowserContext());
```

Comment:
I'm a bit confused about this. Could you explain what's this for? Triggering service creation? How is it related to the tab helper?

Reply:
yes, this is only to trigger service creation, it needs to be done on UI level for browser context